sourcegraph/cody
# Autocomplete: add `CodeGemma` ollama support

## Context
- Adds CodeGemma[link] support for autocomplete via Ollama.

## Test plan
1. Install Ollama https://ollama.com/
2. `ollama pull codegemma:2b` or `ollama pull codegemma:7b`
3. Update user settings:
```
{
  "cody.autocomplete.experimental.ollamaOptions": {
    "url": "http://localhost:11434",
    "model": "codegemma:2b"
  },
  "cody.autocomplete.advanced.provider": "experimental-ollama",
}
```
4. Confirm Cody uses Ollama by looking at the Cody output channel or the autocomplete trace view (in the command palette).
